smoketest/tests/v2/outbound/register_agent.rs

```diff
@@ -1,42 +1,18 @@
-use snowbridge_smoketest::{
-	asset_hub_helper::{eth_location, mint_token_to},
-	contracts::i_gateway_base::AgentCreatedFilter,
-	helper::*,
-	helper_v2::wait_for_ethereum_event_v2,
-	parachains::{assethub, bridgehub::api::ethereum_system_v2::events::CreateAgent},
-};
-use subxt_signer::sr25519::dev;
-
-const INITIAL_FUND: u128 = 3_000_000_000_000;
-const FEE: u128 = 1_000_000_000;
+use ethers::core::types::Address;
+use snowbridge_smoketest::{constants::*, contracts::i_gateway_v2 as i_gateway, helper::*};
 
 #[tokio::test]
 async fn register_agent() {
 	let test_clients = initial_clients().await.expect("initialize clients");
-	let signer = dev::bob();
-
-	// Mint ether to sender to pay fees
-	mint_token_to(
-		&test_clients.asset_hub_client,
-		eth_location(),
-		signer.public_key().0,
-		INITIAL_FUND,
-	)
-	.await;
-
-	let ethereum_system_frontend_api =
-		assethub::api::snowbridge_system_frontend::calls::TransactionApi;
-
-	let call = ethereum_system_frontend_api.create_agent(FEE);
-
-	let _ = test_clients
-		.asset_hub_client
-		.tx()
-		.sign_and_submit_then_watch_default(&call, &signer)
+	let ethereum_client = *(test_clients.ethereum_signed_client.clone());
+	let gateway_addr: Address = (*GATEWAY_PROXY_CONTRACT).into();
+	let gateway = i_gateway::IGatewayV2::new(gateway_addr, ethereum_client.clone());
+	gateway
+		.create_agent(ASSET_HUB_BOB_AGENT_ID)
```

Comment:
We've renamed to `v2_create_agent`.

Reply:
https://github.com/Snowfork/snowbridge/pull/1388/commits/c0a41794885d51b53e796908874338e173dccf8b